Fixing Grammar and Typographical Errors in Documentation (#21357)
MystenLabs/sui: docker/sui-antithesis/README.md

**additional info**

> The version of `sui` which is used to generate the genesis outputs must be on the same protocol version as the fullnode/validators (eg: `mysten/sui-node:mainnet-v1.19.1`)

Reviewer: ```suggestion
The version of `sui` that generates the genesis outputs must be on the same protocol version as the fullnode/validators (for example, `mysten/sui-node:mainnet-v1.19.1`)
```

Author: Done
